enhancements/network/user-defined-network-segmentation.md

```diff
@@ -658,6 +669,7 @@ kind: ClusterUserDefinedNetwork
 metadata:
   name: db-network
 spec:
+  vrf: db-network
```

Comment:
In case VRF field is introduced to provider some integration with other APIs, I suggest having dedicated example for it, including the other APIs it should be integrated with.

Reply:
I don't want to introduce other APIs that then I will have to explain. But I will come up with a user story.